docs/source/guide/prompts_overview.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Prompts overview and use cases- Beta 🧪
+short: Overview and use cases
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 225
+meta_title: Prompts overview
+meta_description: An overview of the Prompts feature on the HumanSignal platform
+section: Prompts
+date: 2024-05-15 14:30:14
+---
+
+
+Use Prompts to evaluate and refine your LLM prompts and then generate predictions to automate your labeling process. 
+
+All you need to get started is an OpenAI API key and a project. 
+
+With Prompts, you can:
+
+* Drastically improve the speed and efficiency of annotations, transforming subject matter experts (SMEs) into highly productive data scientists while reducing the dependency on non-SME manual annotators.
+* Increase annotation throughput, accuracy, and consistency, making the process faster and more scalable. 
+* Empower users to harness the full potential of AI-driven text labeling, setting a new standard for efficiency and innovation in data labeling.
+* Leverage subject matter expertise to rapidly bootstrap projects with labels, allowing you to decrease time to ML development. 
+* Allow your subject matter experts time focus on higher-level tasks rather than being bogged down by repetitive manual work.
+
+
+## Use cases
+
+
+### Auto-labeling with Prompts
+ 
+Prompts allows you to leverage LLMs to swiftly generate accurate predictions, enabling instant labeling of thousands of tasks. 
+
+By utilizing AI to handle the bulk of the annotation work, you can significantly enhance the efficiency and speed of your data labeling workflows. This is particularly valuable when dealing with large datasets that require consistent and accurate labeling. Automating this process reduces the reliance on manual annotators, which not only cuts down on labor costs but also minimizes human errors and biases. With AI's ability to learn from the provided ground truth annotations, you can maintain a high level of accuracy and consistency across the dataset, ensuring high-quality labeled data for training machine learning models.
+
+#### Workflow
+
+1. If you don't already have one, create a project and import a text-based dataset. 
+
+    * [Create a project](setup_project)
+    * [Sync data from external storage](storage)
+2. Annotate a subset of tasks, marking as many as possible as ground truth annotations. The more ground truths you have in your dataset, the more accurate your prompt evaluation results will be. 
```

Comment:
"... the more data you have for prompt evaluation for more confident results"
or smthing like that?
just dont wanna guarantee accuracy